tests/integration/testdata/invoke/template-return-image.yaml

```
@@ -0,0 +1,12 @@
+AWSTemplateFormatVersion : '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: A hello world application.
+
+Resources:
+  GetImageFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: main.image_handler
+      Runtime: python3.9
```

let's use a newer python version
```suggestion
      Runtime: python3.12
```